admin/views/school/school-home.ejs

```diff
@@ -34,7 +34,8 @@
                     <% } %>
                 </li>
             </ol>
-            <h3 class="govuk-heading-m govuk-!-margin-top-9">Preview the check</h3>
+            <!-- TODO JMS Delete this block -->
```

Comment:
superfluous code

Reply:
Yeah - it is at the moment.  It's a draft.